content/concepts/kata/translations.md

```diff
@@ -35,6 +35,10 @@ The description is shared among all language versions so changing it in your tra
 
 The translation does not have to be finished in one go. It can be saved and kept in draft state as long as necessary. When the translator considers it ready, it can be published which places it in a "pending" state and becomes available for other users to review, approve, edit or reject. Only one translation may be published for any given kata and language. For example, if a published Java translation already exists then a new one may be created but not published until the former is rejected.
 
+Unfortunately, currently there's no reliable way to prevent approval of translations of insufficient quality. Unlike beta kata, translations are not a subject to any kind of beta process, and comments in translation discourse cannot be labelled as issues. That's why it's very important to make sure that published translations comply with [translation authoring guidelines (TODO)]().
+
+After translation is published, translator is adviced to post a comment in kata discourse with a link to their translation fork, labelled as **Suggestion**. This will notify other users about the fact and it will be easier for others to find such pending translation.
```

Comment:
```suggestion
After a translation is published, the translator is advised to post a comment in the kata discourse with a link to their translation fork labelled as **Suggestion**. This will notify other users about the fact and it will be easier for others to find such pending translations.
```